packages/react/src/components/Message/MessageHeader.js

```diff
@@ -57,6 +59,8 @@ const MessageHeaderTimestapCss = css`
   color: #9ea2a8;
 `;
 
+const authenticatedUserId = Cookies.get(RC_USER_ID_COOKIE);
```

Comment:
We need to get the authenticated user id from the userStore. We no longer use cookies for storing authenticated user ids.

Reply:
Sure, I’ll make the changes